ui/src/cohort-report-geospatial/styles.css

```diff
@@ -24,6 +24,14 @@ button {
 
 .atlas-cohort-report__btn {
     margin-bottom: 1rem;
+    background-color: #337ab7;
```

Comment:
The style makes it impossible to see the diff between regular and disabled button: include additional CSS for `:disabled` then